avaframe/in3Utils/cfgUtils.py

```diff
@@ -608,7 +608,11 @@ def filterSims(avalancheDir, parametersDict, specDir=''):
             if not isinstance(value, (list, np.ndarray)):
                 value = [value]
             if value != '' and value != []:
-                simDF = simDF[simDF[key].isin(value)]
+                if '~' in key:
+                    keyNew = key.replace("~", "")
+                    simDF = simDF[~simDF[keyNew].isin(value)]
```

Comment:
what is this ~? did not find the origin or signiication

Reply:
this is a new filtering option - to be able to filter according to some value that should not be included

Reply:
done